config/index.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash';
+
+import variantBase from './variants/_base';
+
+export const VARIANTS = [
+  'development',
+  'demo',
+  'production',
```

Comment:
Fair enough. We'll probably still want to make a Webpack `DefinePlugin` value for `process.env.NODE_ENV`, though, to appease weird dependencies like we did in shipper-web. I'm assuming it'll be fine to just use `variant` in that instance.